pages/[user].tsx

```diff
@@ -6,45 +6,61 @@ import { ClockIcon, InformationCircleIcon, UserIcon } from "@heroicons/react/sol
 import prisma from "@lib/prisma";
 import { trpc } from "@lib/trpc";
 import { inferSSRProps } from "@lib/types/inferSSRProps";
-import { GetServerSidePropsContext } from "next";
+import { ssg } from "@server/ssg";
+import { GetStaticPaths, GetStaticPropsContext } from "next";
 import Link from "next/link";
-import React from "react";
+import React, { useEffect } from "react";
 
-export default function User(props: inferSSRProps<typeof getServerSideProps>) {
-  const { isReady } = Theme(props.user.theme);
-  // FIXME delete me before merge
-  const query1 = trpc.useQuery(["test.hello"]);
-  const query2 = trpc.useQuery(["test.hello", "alex"]);
-  console.log("result", { query1, query2 });
+export default function User(props: inferSSRProps<typeof getStaticProps>) {
+  const query = trpc.useQuery(["booking.userAndEventTypes", props.user]);
+  const utils = trpc.useContext();
+  const { isReady } = Theme(query.data?.user?.theme);
+  // prefetch all event types
+  useEffect(() => {
+    if (!query.data) {
+      return;
+    }
+    query.data.eventTypes.forEach(({ slug }) => {
+      utils.prefetchQuery(["booking.eventByUserAndType", { user: props.user, type: slug }]);
+    });
+  }, [props.user, query.data, utils]);
+  if (!query.data) {
+    // this only happens
+    // - in development
+    // - first request to a new user
+    // - 404 pages
+    return "...";
+  }
```

Comment:
This actually shouldn't happen as we do `fallback: 'blocking'` which falls back on SSR